common/build-helper/samba.sh

```diff
@@ -0,0 +1,24 @@
+if [[ $makedepends != *"python3-devel"* ]]; then
+	makedepends+=" python3-devel"
+fi
+
+if [ "$CROSS_BUILD" ]; then
+	# Wrap python-config for samba and its dependencies; this wrapper is
+	# more thorough than the simple default and depends on which
+	if [[ $hostmakedepends != *"which"* ]]; then
+		hostmakedepends+=" which"
+	fi
+
+	cat > ${XBPS_WRAPPERDIR}/python${py3_ver}-config <<-EOF
+	#!/bin/sh
+	exec ${XBPS_CROSS_BASE}/usr/bin/python${py3_ver}-config "\$@"
+	EOF
+
+	chmod 755 ${XBPS_WRAPPERDIR}/python${py3_ver}-config
+	cp -p ${XBPS_WRAPPERDIR}/python{$py3_ver,3}-config
+
+	if command -v qemu-${XBPS_TARGET_QEMU_MACHINE}-static >/dev/null 2>&1; then
+		configure_args+=" --cross-compile --hostcc=${CC_FOR_BUILD}
+		 --cross-execute=qemu-${XBPS_TARGET_QEMU_MACHINE}-static"
+	fi
```

Comment:
I think you can just check for the qemu build helper, which is more semantically correct.